src/compiler/diagnosticMessages.json

```diff
@@ -4683,5 +4683,13 @@
     "Generate types for all packages without types": {
         "category": "Message",
         "code": 95068
+    },
+    "Make all const or readonly expressions reassignable": {

```

Comment:
This sounds like it will change every `const` to `let` regardless of whether there's an error. Maybe this should be something like `Make variables and properties mutable where necessary`.